src/autoComplete/index.unit.tsx

```diff
@@ -343,18 +343,18 @@ describe('AutoComplete', () => {
 
   describe('#focus', () => {
     it('Can add focus to TextField', () => {
-      const wrapper = shallow(<AutoComplete {...defaultProps} focus />)
+      const wrapper = mount(<AutoComplete {...defaultProps} focus />)
```

Comment:
You need to update the `AutoComplete` import that is why your test fails

Reply:
I can't, it's imported from the component Autocomplete itself, it's not imported in the test